### ISSUE TITLE
User Guide/Link Target: Headers Display Under Navbar

### ISSUE THREAD

**Issue by kcondon** [IQSS/dataverse#1520]
This is very minor but I noticed when clicking on the Community Norms link on the licensing tab, the Community Norms label was hidden below the floating nav bar.

**Comment by eaquigley** [comment]
@posixeleni is there a way to change this in Sphinx or is just the way the cookie crumbles?

**Comment by posixeleni** [comment]
@eaquigley this is actually an issue with the nav bar on dataverse.org hiding the part of the page that we anchor to. @mheppler any suggestions ?

**Comment by mheppler** [comment]
@eaquigley @posixeleni -- Not sure how to fix this. I will have to take a look at the Sphinx docs.

**Comment by kcondon** [comment]
Anchor higher up?
________________________________

**Comment by mheppler** [comment]
I am changing the title, and expanding this to cover all links in the guides that come from the app.

**Comment by mheppler** [comment]
Moved this to "Deployment" to go with my other guides-project site-related ticket.

**Comment by mheppler** [comment]
Also seen at:
- [ ] Modification of Terms - http://best-practices.dataverse.org/harvard-policies/harvard-terms-of-use.html#modification-of-terms

**Comment by mheppler** [comment]
Utilized some jQuery (via [Stack Overflow](http://stackoverflow.com/a/20320919)) to fix a UI problem in the guides that caused section headers to display under the navbar.

**Comment by pdurbin** [comment]
@kcondon noticed this issue while testing my pull request at #2895 so I'm planning on testing @mheppler 's fix at b03e359 (1520-guides-navbar-fix branch) on my laptop and potentially merging it into the branch being tested that pull request (2884-install-guide-reorg).

**Comment by pdurbin** [comment]
Locally I switched to the 1520-guides-navbar-fix branch, ran `make clean && make html`, and then tried clicking an anchor from a long list ("Terms") from the User Guide. The bug still seems to be there:

![screen shot 2016-01-29 at 4 03 59 pm](https://cloud.githubusercontent.com/assets/21006/12688120/1aefa60e-c6a2-11e5-838a-14e71eb5a4d9.png)

![screen shot 2016-01-29 at 4 04 32 pm](https://cloud.githubusercontent.com/assets/21006/12688119/1ae90e66-c6a2-11e5-846a-97969233b8ef.png)

I spoke with @mheppler about this and he said to send it to QA. So I am. His fix is still in a branch, 1520-guides-navbar-fix.

**Comment by mheppler** [comment]
@kcondon, when testing this, make sure you give your browser a good "hold SHIFT and refresh" on the guides homepage, before navigating anywhere, so that you clear the old JS and CSS files out of your cache. I suspect that may have been the case with @pdurbin not seeing the fix. I just retested this locally to confirm it was still good. (See attached screenshot following the use case Phil outlined above.)

![screen shot 2016-01-29 at 4 45 07 pm](https://cloud.githubusercontent.com/assets/687227/12688987/ce1d9c2c-c6a7-11e5-9d2b-2af9e048f8c1.png)

**Comment by mheppler** [comment]
I think the problem may have been the need for a `(document).ready` wrapped around the block of code on the navbarscroll.js. Hopefully this fixes it.

**Comment by kcondon** [comment]
@mheppler Still no luck. I decided to run the browser in debug mode while testing and found this error:
TypeError: $(...).on is not a function
$('a[href*=#]:not([href=#])').on('click', function() {
	
navbarscroll.js (line 6, col 3)

**Comment by mheppler** [comment]
After digging through some [documentation for the Sphinx-Bootstrap theme](https://ryan-roemer.github.io/sphinx-bootstrap-theme/README.html?highlight=jquery) I was able to identify that the new jQuery I added in the `navbarscroll.js` was using the incorrect `$` shortcut identifier that the theme was setup to use, which was what was causing the `TypeError` and breaking the scroll animation.

This has been fixed with a "trial and error" approach of committing fixes to the dataverse-internal test environment, set up with the help of @kcondon. The fixed functionality has been since in both Firefox and Chrome on my machine and confirmed by @pdurbin on his.
